Content.Server/Atmos/EntitySystems/FlammableSystem.cs

```diff
@@ -209,11 +209,7 @@ public override void Update(float frameTime)
 
                 if (flammable.FireStacks > 0)
                 {
-                    _temperatureSystem.ReceiveHeat(uid, 200 * flammable.FireStacks);
-                    // TODO ATMOS Fire resistance from armor
-                    var damageScale = Math.Min((int) (flammable.FireStacks * 2.5f), 10);
-                    _damageableSystem.TryChangeDamage(uid, flammable.Damage * damageScale);
-
+                    _temperatureSystem.ChangeHeat(uid, 80000 * flammable.FireStacks);
```

Comment:
uhhhhh that's a rather insane temperature.

Reply:
its not a temperature, it's thermal energy; this gets divided by the heat capacity of the mob later, which brings it to a reasonable temperature (about a 30-60 degree increase per second)

Reply:
heat capacity is determined by mass and specific heat so if you lit a mouse on fire it would die way faster than a human. fun fact

Reply:
I think there's an issue here with a mob's internal heat regulation. I tested the flammable damage by applying fire stacks and setting on-fire via VV. Some results were:
3 fire stacks -> walk around on fire for 30 seconds -> sweat a bit -> take 0.32 burn damage
5 fire stacks -> 50 seconds -> ~17 burn damage
8 fire stacks -> 80 seconds -> ~85 burn damage

I'm not sure what the right numbers are for balance purposes, but any degree of being on fire should deal SOME damage. Currently less than 3 stacks will deal none at all. IMO, no amount of sweating should ever be able negate literally being on fire. 

Without making fire-stacks deal direct burn damage, I guess you could add a constant to the heat formula, so that even 0.01 fire stacks can overwhelms sweating? Basically the sum of `ImplicitHeatRegulation + SweatHeatRegulation` for default humans? That, or somehow disable sweating while on fire? I'm not really sure how best to simulate being on fire without double-dipping on heating & applying burn directly.

Balance wise I think that if you're continuously on fire for 30 seconds you should probably be in crit or close to it. A 2 second stop-drop-n-roll shaves off ~48 fire-seconds / 5 stacks (3 stacks at the end, and while rolling it goes down at 1-stack per second instead of 0.1). So I think that's not too unreasonable?